frontend/src/components/Map/Location.tsx

```diff
@@ -39,18 +39,21 @@ interface LocationState {
   hover: boolean;
 }
 
+// logarithmic scale for location dots
+const getNodeSize = (nodeCount: number) => {
+  const log = Math.log(nodeCount) / Math.log(2);
+  const size = 6 + log * 2;
+  return size < 1 ? 1 : size;
+};
+
 export class Location extends React.Component<LocationProps, LocationState> {
   public readonly state = { hover: false };
 
   public render() {
     const { position, nodeCount } = this.props;
     const { left, top, quarter } = position;
     const className = `Location Location-quarter${quarter}`;
-
-    // location dots have logarithmic scale
-    // 6px is default size for a single node dot,
-    // log of 1 is 0, so we have fallback value of 1
-    const size = 6 * (Math.log(nodeCount) || 1);
+    const size = getNodeSize(nodeCount);
```

Comment:
I think size here is always >= 6. I think just setting `const size = 6 + Math.log(nodeCount) * 2` will do the job, as `nodeCount >= 1`, which implies that `Math.log(nodeCount) >= 0`, which implies that `size = 6 + Math.log(nodeCount) * 2 >= 6`